docs/data/toolpad/getting-started/introduction.md

```diff
@@ -0,0 +1,24 @@
+# Introduction
+
+<p class="description">Quickly build internal tools. MUI Toolpad is a self-hosted low-code internal tools builder designed to extend the MUI React components. It's designed for developers of all trades who want to save time building internal applications.</p>
+
+## What is Toolpad?
+
+Learn a few core concepts of the Toolpad app and you are ready to ship your application in minutes! Drag & drop pre-built UI components, connect your data sources, release your app - and you're done!
+
+![Editor screenshot](/static/toolpad/marketing/hero-screenshot.png)
+
+## How Toolpad works?
+
+- [Build UI](/toolpad/core-concepts/building-ui/) using our drag & drop editor
+- [Configure and fetch](/toolpad/core-concepts/connecting-to-data-sources/) data from HTTP endpoints, Google sheets or even directly from Postgres SQL DB
```

Comment:
[PostgreSQL](https://www.postgresql.org/) seems to refer to themselves as "PostgreSQL" on their website. Updating it in the app as well https://github.com/mui/mui-toolpad/pull/894

```suggestion
- [Configure and fetch](/toolpad/core-concepts/connecting-to-data-sources/) data from HTTP endpoints, Google sheets or even directly from PostgreSQL
```

Reply:
From wiki: https://en.wikipedia.org/wiki/PostgreSQL

> PostgreSQL also known as Postgres...

I guess both should be fine